data/creatures/bubblegum.yaml

```diff
@@ -74,6 +73,7 @@
   Level: 3
   Abilities:  
     - "Once per turn you may deal up to 6 damage to Blastilo. Then INCREMENT up to the amount of damage Blastilo took."
+    - "Instead of INCREMENT 1 Blastilo may gain 1 heart."
```

Comment:
It might be confusing because it can be thought of in two ways:
1. If you can Increment up to 5, you can increment 4 times and then also get 1 heart up.
2. Only when you have exactly Increment 1, you can instead gain 1 heart. 
I assume you imagined the first way.
Maybe should be rephrased to: "Blastilo may Increment by 1 less to gain 1 heart."

Reply:
went with the 2. option